Web/RotaMe.Web/Areas/Admin/Controllers/RolesController.cs

```diff
@@ -42,20 +74,38 @@ public async Task<IActionResult> List()
                 {
                     case "Users":
                         listRoleViewModel[i].CardIcon = "fa-user";
+                        listRoleViewModel[i].DeleteButton = false;
                         break;
                     case "Admins":
                         listRoleViewModel[i].CardIcon = "fa-user-tie";
+                        listRoleViewModel[i].DeleteButton = false;
                         break;
                     case "Owners":
                         listRoleViewModel[i].CardIcon = "fa-user-alt";
+                        listRoleViewModel[i].DeleteButton = false;
                         break;
                     default:
                         listRoleViewModel[i].CardIcon = "fa-user";
+                        listRoleViewModel[i].DeleteButton = true;
                         break;
                 }
             }
 
             return View(listRoleViewModel);
         }
+
+        [HttpPost]
+        public async Task<IActionResult> Delete(string id)
+        {
+
+            var isRoleDeleted = await rolesService.Delete(id);
+
+            if (isRoleDeleted)
+            {
+                return this.StatusCode(200);
+            }
+
+            return this.StatusCode(400);
```

Comment:
thats a good example how you could have used if-else but you did it the correct way here... Be consistent in your code